_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,361 @@
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19305][] adds release notes for the upcoming codebase
+  transition from the C++11 standard to C++17. The plan is to add C++17
+  compatibility in the 0.21 release ([expected][Bitcoin Core #18947] end
+  of 2020) and break C++11 compatibility later in 0.22 (expected mid-2021).
+  Although this change won't affect most users, those planning to build
+  the 0.22 release on older systems may want to check their toolchain's standard
+  support against the [C++ compiler support matrix][cpp compiler support].
+
+- [Bitcoin Core #11413][] updates the `bumpfee`, `fundrawtransaction`,
+  `sendmany`, `sendtoaddress`, and `walletcreatefundedpsbt` RPCs to
+  allow manually specifying the feerate to use in the newly created or
+  updated transaction.  By default, these commands still use either an
+  automatic feerate computed by Bitcoin Core's built-in transaction fee
+  estimate or the fallback feerate if there's not enough data for
+  estimation.  For details on how to use the updated RPCs, see the
+  [proposed release note][bcc11413rn].  Despite not affecting any major
+  system, this PR was open for almost three years---the second longest
+  of any PR that has been merged into Bitcoin Core so far.  We thank the
+  author, Kalle Alm, and everyone else involved for their persistence.
+
+- [Eclair #1466][] decreases the length of time that an Eclair node will wait
+  before closing a channel when a peer becomings unresponsive while an HTLC is
+  pending. To prevent payments from becoming stuck forever, each HTLC includes
+  a timelock. If the timelock expires, then the sending party can reclaim
+  the funds in the HTLC. To prevent the sending party from pulling back
+  funds that an intermediate party has already paid to the next hop, the
+  intermediate party must settle the pending HTLC onchain if the sending
+  party becomes unresponsive. This PR increases Eclair's safety window---it
+  will now broadcast the on chain transaction 24 blocks before the HTLC
+  timeout, instead of 6 blocks before the HTLC timeout.
+
+- [LND #4018][] adds the ability to delay forwarding a payment (HTLC),
+  giving an external process the ability to review it and decide whether
+  to cancel it or continue forwarding it.  This is similar to [hold
+  invoices][topic hold invoices] but it applies to payments being routed
+  to a node's peers rather than received by the node itself.  Several
+  use cases are described in the PR, such as being able to hold the
+  HTLC, detect that its next hop is offline (e.g. a mobile node), send a
+  notification to that node which will restart it, and then continue
```

Comment:
perhaps `s/, send/, or send/` combined with `s/it, and/it and/` to help readers understand that from this point we are describing a single last use case (if I'm understanding the writing correctly).